src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -119,7 +119,7 @@ source %{
         ShouldNotReachHere();
       }
       int imm4 = disp / mesize / Matcher::scalable_vector_reg_size(vector_elem_bt);
-      (masm.*insn)(reg, Assembler::elemType_to_regVariant(vector_elem_bt), pg, Address(base, imm4));
+      (masm->*insn)(reg, Assembler::elemType_to_regVariant(vector_elem_bt), pg, Address(base, imm4));
```

Comment:
this update is missing in the corresponding `aarch64_vector_ad.m4` file.
(that is, there is a mismatch between the ad file and the generated one.)